python/python/glide/async_commands/cluster_commands.py

```diff
@@ -742,3 +744,83 @@ async def random_key(self, route: Optional[Route] = None) -> Optional[str]:
             Optional[str],
             await self._execute_command(RequestType.RandomKey, [], route),
         )
+
+    async def scan(
+        self,
+        cursor: ClusterScanCursor,
+        match: Optional[str] = None,
```

Comment:
I know it is different than how i used it in other places, i used object type since type is a problematic word, as well as match in Rust, so i couldn't use it, but the server API use match and type as the parameters and i want to provide the same to the client.